Sources/BlocksRuntime/runtime.c

```diff
@@ -268,7 +268,11 @@ void _Block_use_RR( void (*retain)(const void *),
         break;
     }
 #else
+# if __has_include(<dlfcn.h>)
     _Block_destructInstance = dlsym(RTLD_DEFAULT, "objc_destructInstance");
+# else
+    assert(0 && "No way to find objc_destructInstance on this platform");
```

Comment:
Can we just set `_Block_destructInstance` to `NULL`? IIRC, `_Block_destructInstance(...)` should do nothing if the pointer is NULL (this is how weak pointers work).

Reply:
`_Block_destructInstance` can be called without null guard https://github.com/apple/swift-corelibs-foundation/blob/dbca8c7ddcfd19f7f6f6e1b60fd3ee3f748e263c/Sources/BlocksRuntime/runtime.c#L530, so `_Block_destructInstance_default` might be safer here